vagrant/Vagrantfile

```diff
@@ -4,7 +4,8 @@
 # For a complete reference, please see the online documentation at https://docs.vagrantup.com.
 Vagrant.configure("2") do |config|
 
-  config.vm.box = "boxcutter/centos7-desktop"
+  config.vm.box = "centos7-desktop"
+  config.vm.box_url = "/Users/bratseth/development/github/vespa-engine/vespa/vagrant/centos7-desktop-17.0915.1.box"
```

Comment:
I think this will not work for most users..! 😉

Reply:
Thanks. Since we don't have a working solution atm I just put TODO.